src/types.ts

```diff
@@ -1,3 +1,4 @@
+import { URLSearchParams } from 'url'
```

Comment:
What is this used for?

Reply:
@walleXD Take a look at the end of this file